src/Page/Form.purs

```diff
@@ -191,8 +150,24 @@ component =
       }
       [ renderInput field ]
 
-  renderInput :: Field ExprType -> H.ComponentHTML Query
+  renderInput :: Field ExprType -> H.ParentHTML Query (ChildQuery m) ChildSlot m
   renderInput (field) = case field.input of
+    Dropdown dropdown ->
+      HH.slot'
+        cp1
+        field.key
+        Dropdown.component
+        { selectedItem: getValue dropdown
+        , items: fold (toArray dropdown.options)
```

Comment:
Have we tested that when the `options` change, that's reflected in the component? We might need to send a query down to the component telling it to update its items.

Reply:
Which, would seem like a design flaw really...

Reply:
Nope, no tests for that. How do we tend to test Halogen components?

Reply:
Actually, what is it that we lack confidence in? Is it our eval, `ocelot`, or `halogen`? The first seems much easier to validate than the other two. The other two also don't seem like they should need to be validated in `lynx`.

Reply:
So, if I remember correctly, updates to `dropdown.options` in the parent state, don't trigger a new `input` being sent to the child component, we have to manually do it through a query in our parent `eval`. If that's the case it's a Halogen thing, but the design flaw could be considered either in Halogen or Ocelot. Or both.

Reply:
So I manually tested this with PR #9, and they don't update.

Reply:
Your PR is proof that this is a design flaw (and it's bitten us before as well), because the behavior isn't expected. I feel Ocelot should be updated with a `receiver` to handle updates to state

Reply:
Implemented this in https://github.com/citizennet/purescript-lynx/pull/8/commits/c2274913c2ddb22a786034ab112de66a7c195a5b